src/zip/fs.ts

```diff
@@ -293,16 +314,7 @@ const _Zip = {
 	name: 'Zip',
 
 	options: {
-		data: {
-			type: [
-				ArrayBuffer,
-				Object.getPrototypeOf(Uint8Array) /* %TypedArray% */,
-				function ZipDataSource(v: unknown): v is ZipDataSource {
-					return typeof v == 'object' && v !== null && 'size' in v && typeof v.size == 'number' && 'get' in v && typeof v.get == 'function';
-				},
-			],
-			required: true,
-		},
+		data: { type: 'object', required: true },
```

Comment:
I simplified this check, as it is done in `SimpleBufffer` backend, this check fails in some Unit Test scenarios, complaining `ArrayBuffer` was not used with `new`. I think is an overkill to check this way, in particular if the consumer uses TypeScript.

Reply:
Sounds good.